draft-ietf-acme-acme.md

```diff
@@ -768,6 +762,8 @@ syntax like "confirm": true ]]
 support reissuance / renewal in a straightforward way, especially if the CSR /
 notBefore / notAfter could be updated. ]]
 
+#### Application Requirements
+
```

Comment:
Note: These changes are unrelated to the topic of the PR.  I think they're OK, though.